ethereum/datastructures/src/main/java/tech/pegasys/teku/datastructures/blocks/BeaconBlock.java

```diff
@@ -42,6 +45,9 @@
   // Body
   private BeaconBlockBody body;
```

Comment:
Looks like we should also do this for `BlockBody`

Reply:
Makes sense.